### ISSUE TITLE
fedmsg-logger --json-input can't handle multiline json

### ISSUE THREAD

**Issue by glensc** [fedora-infra/fedmsg#372]
seems `--json-input` can't handle multiline json (which is valid json), likely due parser is line based, but should read all input before attempting json decode:

```
[~/scm/fedmsg] ➔ cat git.json|/usr/bin/fedmsg-logger --json-input --modname git --topic receive 
Traceback (most recent call last):
  File "/usr/bin/fedmsg-logger", line 9, in <module>
    load_entry_point('fedmsg==0.16.4', 'console_scripts', 'fedmsg-logger')()
  File "/usr/share/python2.7/site-packages/fedmsg/commands/logger.py", line 114, in logger
    return command.execute()
  File "/usr/share/python2.7/site-packages/fedmsg/commands/__init__.py", line 104, in execute
    return self.run()
  File "/usr/share/python2.7/site-packages/fedmsg/commands/logger.py", line 108, in run
    self._log_message(self.config, line.strip())
  File "/usr/share/python2.7/site-packages/fedmsg/commands/logger.py", line 85, in _log_message
    msg = fedmsg.encoding.loads(message)
  File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting object: line 1 column 12 (char 11)
[~/scm/fedmsg] ➔ cat git.json|tr '\n' ' '|/usr/bin/fedmsg-logger --json-input --modname git --topic receive 
[~/scm/fedmsg] ➔ 
```

**Comment by krisananthu** [comment]
I'd like to fix this. Could you tell how to built it locally

**Comment by krisananthu** [comment]
Is the  (fedmsg)$ pip install -e .[all] a terminal command or is the [all] supposed to be replaced?

**Comment by jeremycline** [comment]
Should be fixed in #392
